### PR TITLE
Updated Filters Panel UI

### LINKED ISSUE #190: Color Filter Names In Panel
Right now the names are not colored, and this makes it difficult to tell which colors indicate which tokens. The names and highlighting should be the same color.

### PR DESCRIPTION
**Pull request by Hopding**
Fixes #190. This PR removes the checkboxes from the filters panel, and replaces them with clickable labels, whose color toggles on/off when clicked. Screenshots below:

## No Labels Selected
screen shot 2017-03-15 at 7 52 06 pm

## Labels Selected (Large Width)
screen shot 2017-03-15 at 7 52 29 pm

## Labels Selected (Medium Width)
screen shot 2017-03-15 at 7 52 39 pm

## Labels Selected (Small Width)
screen shot 2017-03-15 at 7 52 49 pm

### CODE REVIEW

**Comment by thebho** [comment]
Oh, it's definitely cleaner.  Let's not merge this until we've discussed it

**Comment by dane-johnson** [comment]
Possible addition. Make the count a grey, right aligned span in the button instead of being tacked on to the name.

**Comment by Hopding** [comment]
@dane-johnson This is what that looks like (I like the black text more):

## Black Text
screen shot 2017-03-16 at 5 48 04 pm

## Greyish Text
screen shot 2017-03-16 at 5 52 08 pm

File: src/containers/AppBody.jsx [hunk 1]
```diff
@@ -35,15 +36,15 @@ export class AppBody extends React.Component {
...
+          <div className='col-lg-4 col-md-7'>
             <Card>
               <SnippetArea
                 id={id}
               />
             </Card>
           </div>
           <div className='col-lg-5 col-md-5'>
-            <Card><Annotations /></Card>
+            <Card><Annotations id={id} /></Card>
```
Comment (solkaz) [inline src/containers/AppBody.jsx]: There a need to pass the id to Annotations?

Comment (Hopding) [inline src/containers/AppBody.jsx]: Not sure how that got there.

**Pull request merged**